pkg/datapath/linux/bigtcp/bigtcp.go

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package bigtcp
+
+import (
+	datapathOption "github.com/cilium/cilium/pkg/datapath/option"
+
+	"github.com/vishvananda/netlink"
+
+	"github.com/cilium/cilium/pkg/datapath/linux/probes"
+	"github.com/cilium/cilium/pkg/option"
+)
+
+const (
+	defaultGROMaxSize int = 65536
+	defaultGSOMaxSize int = 65536
+
+	BIGTCPGROMaxSize int = 196608
+	BIGTCPGSOMaxSize int = 196608
```

Comment:
It's more common in Go to declare numeric constants as untyped. That way they can be used with arbitrary integer types depending on context. Any particular reason these need to be `int`s? I think making them untyped would allow to get rid of the type conversions in `setGROGSOMaxSize` below.

```suggestion
	defaultGROMaxSize = 65536
	defaultGSOMaxSize = 65536

	BIGTCPGROMaxSize = 196608
	BIGTCPGSOMaxSize = 196608
```

Reply:
no particular reason or preference, I'll change them

Reply:
./bigtcp.go:63:26: invalid operation: attrs.GROMaxSize == GROMaxSize (mismatched types uint32 and int)
./bigtcp.go:63:60: invalid operation: attrs.GSOMaxSize == GSOMaxSize (mismatched types uint32 and int)
./bigtcp.go:65:24: invalid operation: attrs.GROMaxSize <= GROMaxSize (mismatched types uint32 and int)
./bigtcp.go:66:24: invalid operation: attrs.GSOMaxSize <= GSOMaxSize (mismatched types uint32 and int)

I'll rather keep them explicitly as ints.

Reply:
Right, the type conversions in setGROGSOMaxSize are still needed then. But I think the consts can remain untyped.

Reply:
Yep, they can. I'll push the changes in a minute